module/zfs/dsl_crypt.c

```diff
@@ -1952,6 +1970,12 @@ dsl_crypto_recv_key_check(void *arg, dmu_tx_t *tx)
 		goto error;
 	}
 
+	ret = nvlist_lookup_uint64(nvl, DSL_CRYPTO_KEY_VERSION, &version);
+	if (ret != 0 || version != ZIO_CRYPT_KEY_CURRENT_VERSION) {
+		/* don't receive old on-disk formats */
```

Comment:
Please include an explanation of why this old format can't be received.  Referencing the errata would be a reasonable way to handle this.